torch/fft/__init__.py

```diff
@@ -815,7 +858,7 @@
 """.format(**factory_common_args))
 
 rfftfreq = _add_docstr(_fft.fft_rfftfreq, r"""
-rfftfreq(n, d=1.0, *, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
+rfftfreq(n, d=1.0, *, out=None, dtype=None, layout=torch.strided, device=None, requires_grad=False, out=None) -> Tensor
```

Comment:
I get that we had too few out= kwargs previously, but isn't this going overboard? :P